docs/Examples/Basic.md

```diff
@@ -25,32 +26,38 @@ extension MyService: TargetType {
         switch self {
         case .zen:
             return "/zen"
-        case .showUser(let id):
+        case .showUser(let id), .updateUser(let id, _, _):
             return "/users/\(id)"
         case .createUser(_, _):
             return "/users"
         case .showAccounts:
-        	  return "/accounts"
+            return "/accounts"
         }
     }
     var method: Moya.Method {
         switch self {
         case .zen, .showUser, .showAccounts:
             return .get
-        case .createUser:
+        case .createUser, .updateUser:
             return .post
         }
     }
     var parameters: [String: Any]? {
         switch self {
         case .zen, .showUser, .showAccounts:
             return nil
-        case .createUser(let firstName, let lastName):
+        case .createUser(let firstName, let lastName), .updateUser(_, let firstName, let lastName):
             return ["first_name": firstName, "last_name": lastName]
         }
     }
+
     var parameterEncoding: ParameterEncoding {
-        return URLEncoding.default
+        switch self {
+        case .zen, .showUser, .showAccounts, .updateUser(_, _, _):
```

Comment:
This is a minor one, but I think we can strip these `(_, _, _)` when we don't need any of the parameters in there.